diagnostic_feedback/templates/studio/wizard.html

```diff
@@ -0,0 +1,97 @@
+{% load i18n %}
+<div class="diagnostic-feedback diagnostic-feedback-studio">
+    <div class="editor">
+        <div class="wrapper-comp-settings is-active editor	">
+            <h1 class="wizard-loading">{% trans "Loading Wizard, Please wait" %}</h1>
+
+            <div id="edit_questionnaire_panel" class="hide">
+                <div class="msg"><h3></h3></div>
+                <form id="questionnaire-form" action="#">
+                    <div>
+                        <!-- Step 1 start -->
+                        <h3>{% trans "Basic Info" %}</h3>
+                        <section step="1">
+                            <hr/>
+                            <ul class="list-input settings-list">
+                                <li class="field comp-setting-entry metadata_entry">
+                                    <div class="wrapper-comp-setting">
+                                        <label for="title" class="label setting-label">{% trans "Quiz Title" %}</label>
+                                        <input id="title" name="title" type="text" class="required field-data-control"
+                                               required value="{{self.title}}">
+                                    </div>
+                                </li>
+                                <li class="field comp-setting-entry metadata_entry">
+                                    <div class="wrapper-comp-setting">
+                                        {% if not self.quiz_type %}
+                                        <label for="type" class="label setting-label">{% trans "Quiz Type" %}</label>
+                                        <select id="type" name="type" class="required field-data-control" required>
+                                            <option value="">{% trans "Select Type" %}</option>
+                                            {% for t in self.types %}
+                                            <option value="{{t.value}}" {% if t.value == self.quiz_type %}
+                                                    selected="selected" {% endif %}>
+                                                {{t.label}}
+                                            </option>
+                                            {% endfor %}
+                                        </select>
+                                        {% else %}
+                                        <p class="label setting-label">{% trans "Quiz Type" %}
+    {% for t in self.types %} {% if t.value == self.quiz_type %} {{t.label}} {% endif %} {% endfor %}</p>
```

Comment:
Thanks, I think this is nicer :)

Can you please move the value ("BuzzFeed-style Quiz" etc.) outside of the `<p class="label">` so it gets aligned with the "Quiz title" field above? Right now, the label and the value are jumbled together.

Before:
![screen shot 2015-11-30 at 5 48 15 pm](https://cloud.githubusercontent.com/assets/945577/11490150/cb60c7a8-978a-11e5-9a5b-173731d50421.png)

Fixed:
![screen shot 2015-11-30 at 5 50 16 pm](https://cloud.githubusercontent.com/assets/945577/11490155/d66c144a-978a-11e5-9b91-26912015f0cf.png)